server/planning/search/queries/combined.py

```diff
@@ -22,8 +23,12 @@ def construct_combined_view_data_query(
 
     query = elastic.ElasticQuery()
 
-    if len(search_filter["params"]):
-        search_dates(search_filter["params"], query)
+    filter_params = common.get_params_from_search_filter(search_filter)
+    if len(filter_params):
+        filter_params["time_zone"] = params.get("time_zone") or app.config.get("DEFAULT_TIMEZONE")
+        filter_params["start_of_week"] = params.get("start_of_week") or app.config.get("START_OF_WEEK", 0)
```

Comment:
this might be tricky if we start using the params for it and there will be `0`, rather do it like:
```py
params.get("start_of_week", app.config.get("START_OF_WEEK", 0))
```